packages/material-ui/src/locale/index.ts

```diff
@@ -1,4 +1,36 @@
-export const azAZ = {
+interface Localization {
+  props?: {
+    MuiBreadcrumbs?: {
+      expandText?: string;
+    };
+    MuiTablePagination?: {
+      backIconButtonText?: string;
+      labelRowsPerPage?: string;
+      labelDisplayedRows?: (data: { from: number; to: number; count: number }) => string;
```

Comment:
Should we import the types from the components' props?

Reply:
Is this how these are used? They're just merged into the theme?

Reply:
If we take this prop as an example, `azAZ.props.MuiTablePagination.labelDisplayedRows` will go into `theme.props.MuiTablePagination.labelDisplayedRows`, that will late change the default value of `<TablePagination labelDisplayedRows/>`. Yes, they are merged into the theme without any transformations.

Reply:
Great suggestion. Already uncovered missing prop types :)

Reply:
But the types get a little bit more complicated. Hopefully this isn't too complicated as an entry level.